src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
Hm, removing the copy constructor and assignment operators that took a `Connection` seems to have exposed an issue with `ScopedConnection`'s move assignment operator. Basically the old (moved) ScopedConnection ended up still disconnecting the signal. Tentative fix [here](https://github.com/cinder/Cinder/pull/1676/commits/c5cac01d293b9a0a210ba263e13327acac97d0aa) (with a new unit test for it as well).